web/types/next-auth.d.ts

```diff
@@ -44,6 +44,7 @@ declare module "next-auth" {
       projects: {
         id: PrismaProject["id"];
         name: PrismaProject["name"];
+        deletedAt: PrismaProject["deletedAt"];
```

Comment:
do we have to add this here? Id prefer to not make any changes on next-auth logic.

Reply:
@maxdeichmann This seems to be the information that we use in the frontend landing page to decide how to render the projects. I.e. to make the change in `web/src/features/organizations/components/ProjectOverview.tsx` this seemed like the place I had to edit.